provider/vsphere/instance.go

```diff
@@ -53,11 +53,7 @@ func (inst *environInstance) Addresses() ([]network.Address, error) {
 	res := make([]network.Address, 0)
 	for _, net := range inst.base.Guest.Net {
 		for _, ip := range net.IpAddress {
-			scope := network.ScopeCloudLocal
-			if net.Network == inst.env.ecfg.externalNetwork() {
```

Comment:
Do we want to keep this logic still? If VMWare allows you to configure say, a 10. range as your external network (which makes sense for an internal setup perhaps), we'd currently be deriving that as CloudLocal when perhaps Public is more appropriate?